.github/pull_request_template.md

```diff
@@ -14,14 +15,14 @@ If this pull request adds a new package:
   - [ ] a valid logo URL that points to a PNG whose dimensions conform to the others in this repo (e.g., 100x100).
 - [ ] The package repo contains an Overview doc (`/docs/_index.md`) that includes:
   - [ ] a brief explanation of what the package is and what it does.
-  - [ ] at least one representative example in all supported languages.
+  - [ ] at least one representative example in all supported languages (ie. ts/js, python, go and c#).
```

Comment:
What about Java? this would still retain a discrepancy between all official Pulumi languages and what's required here.